.github/workflows/tests.yml

```diff
@@ -41,16 +41,19 @@ jobs:
       matrix:
         destination:
           - platform=macOS
-          - platform=iOS Simulator,OS=latest,name=iPhone 11
-        skip-testing: [ 'TestProjectTests/FailingTests' ]
+          - platform=iOS Simulator,OS=latest,name=iPhone 13
+        skip-testing:
+            - |
+                TestProjectTests/FailingTests/testFailing1()
+                TestProjectTests/FailingTests/testFailing2()
```

Comment:
Actually, they are needed up until Xcode 13. I've had failures [here](https://github.com/sersoft-gmbh/xcodebuild-action/runs/3740800181?check_suite_focus=true) without them. But I've removed them because I've also updated the tests to use Xcode 13, where the brackets are no longer needed.